src/components/WorkshopList.astro

```diff
@@ -0,0 +1,85 @@
+---
+import { graphql } from "gql.tada";
+import { client } from "../utils/graphql-client";
+import { WORKSHOP_YEAR_TAG_KEY } from "../utils/constants";
+import SpeakerCard, { speakerCardFragment } from "./SpeakerCard.astro";
+import { speakerDetailFragment } from "./SpeakerDetail.astro";
+import { notEmpty } from "../utils/helpers";
+
+Astro.response.headers.set(
+	"Cache-Control",
+	"s-maxage=60, stale-while-revalidate"
+);
+
+export interface Props {
+	slug: string;
+	class?: string;
+}
+
+const { slug, class: className, ...rest } = Astro.props;
+
+const query = graphql(
+	`
+		query GetWorkshops($limit: Int!, $yearTag: String!) {
+			workshops: workshopCollection(
+				limit: $limit
+				where: { tags: { title: $yearTag } }
```

Comment:
Looks like we can simplify these queries, no need to query `tagCollection` and `linkedFrom` anymore.